R/module_teal.R

```diff
@@ -184,7 +186,33 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
       }
     )
 
-    data_rv <- srv_init_data("data", data = data, modules = modules, filter = filter)
+    init_data <- srv_init_data("data", data = data, modules = modules, filter = filter)
+    data_validate <- reactive(tryCatch(init_data(), error = function(e) e))
```

Comment:
Hey @vedhav this variable is never used `data_validate`

Reply:
```suggestion
```
Thanks Marcin! It was from a previous implementation.